docker-compose.yml

```diff
@@ -0,0 +1,10 @@
+services:
+  company-node-app:
+    build: .
+    image: 422-final:latest
+    container_name: 422-final
+    volumes:
+      - .:/app/inbound
+      - .:/app/outbound
+      - .:app/processed
```

Comment:
This would co-mingle the files. You probably would want the folders defined in the prefix, no?